corebehrt/common/utils.py

```diff
@@ -100,80 +53,109 @@ def load_tensor(filename, required=False):
                 else:
                     return None
             return torch.load(filepath)
-        features = load_tensor(f'{mode}_features.pt', required=True)
-        pids = load_tensor(f'{mode}_pids.pt', required=True)
-        outcomes = load_tensor(f'{mode}_outcomes.pt')
-        censor_outcomes = load_tensor(f'{mode}_censor_outcomes.pt')
-        vocabulary = load_tensor('vocabulary.pt')
+
+        features = load_tensor(f"{mode}_features.pt", required=True)
```

Comment:
I think we should check if filepath exists and load if exists (using `path().exists()`) rather than "loading" a file that isn't present for clarity

Reply:
Do you mean doing a check inside the load_from_directory method instead of inside the submethod? 
Since not all the files need to be present to construct the data, we check if optional files are in the directory and return None if not. What would you suggest instead?

We might figure out a better way to do it for the new dataclass.

Reply:
Yeah there might be a few options to "fix" it IMO. Just feel like it's a little odd to have a bunch of load statements that might be doing
We might do it inside the load_from_directory method or maybe we can just change the `load_tensor` to `load_if_exists` or something like that to clarify. Or have a pretrain/finetune one, so we only load the expected files or something like that